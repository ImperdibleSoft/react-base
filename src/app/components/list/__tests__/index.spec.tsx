import * as React from 'react';
import { shallow } from 'enzyme';

import List from '..';

describe('List', () => {
  it('should render a valid list', () => {
    const component = shallow(
      <List>
        {({ Item }) => (
          <>
            <Item>List item</Item>
          </>
        )}
      </List>
    );

    expect(component).toMatchSnapshot();
  });
});
