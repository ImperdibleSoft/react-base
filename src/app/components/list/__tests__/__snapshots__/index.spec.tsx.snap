// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List should render a valid list 1`] = `
<List>
  <Item>
    List item
  </Item>
</List>
`;
